app/buck2_re_configuration/src/lib.rs

```diff
@@ -352,6 +356,14 @@ impl Buck2OssReConfiguration {
                     property: "use_fbcode_metadata",
                 })?
                 .unwrap_or(true),
+            max_decoding_message_size: legacy_config.parse(BuckconfigKeyRef {
+                section: BUCK2_RE_CLIENT_CFG_SECTION,
+                property: "max_decoding_message_size",
+            })?,
+            max_total_file_size: legacy_config.parse(BuckconfigKeyRef {
```

Comment:
I guess I don't mind doing this, but can you give an example of why a user might want to configure this away from the default? (also for the `max_decoding_message_size` I guess)

Reply:
There are some advantages to batching more or less and some of them depend on the implementation of the RE protocol. An implementation that does not delegate uploads to a distributed system may have hotspotting if you tried to upload too many files in parallel and thus such a limit would help control this issue (though if would also be useful to add a `max_files_per_batch` limit for the same case). On the other hand, if this isn't the case it is generally advantegeus to use as big of a batch as you can in order to reduce the back and forth of every method call. `max_decoding_message_size` follows that very same logic, with the added constraint that a large portion of the message might be coming from the digests, which we do not account for (this is especially bad for builds that generate lots and lots of small files). So if you can change the `max_files_per_batch` then you should also be able to increase the message size accordingly. I don't think either flags will be changed often, but in case this is needed it is rather handy to be able to. The default behaviour is good enough but we shouldn't shy away from allowing tweaking